src/ui/tabs/rental_info_tab.py

```
@@ -207,15 +213,43 @@ def upload_image(self, image_type):
             if not self._validate_image_file(file_path):
                 QMessageBox.warning(self, "Invalid File", "The selected file is not a valid image.")
                 return
+
+            # Ensure the image storage directory exists
+            self.IMAGE_STORAGE_DIR.mkdir(parents=True, exist_ok=True)
+
+            try:
+                # Generate a unique filename to avoid collisions
+                original_filename = Path(file_path).name
```

Ruff (0.11.9)

Local variable `original_filename` is assigned to but never used

Remove assignment to unused variable `original_filename`

(F841)